src/stdatamodels/fits_support.py

```diff
@@ -449,7 +449,7 @@ def _fits_keyword_loader(hdulist, fits_keyword, schema, hdu_index, known_keyword
     return val
 
 
-def _fits_array_loader(hdulist, schema, hdu_index, known_datas):
+def _fits_array_loader(hdulist, schema, hdu_index, known_datas, context):
```

Comment:
Is there a reason all these functions need to pass around a context, as opposed to just being methods of the parent class that already have access to `self`?

Reply:
That's a good idea, we're overloading the term "context" in this module (sometimes it's the AsdfFile object, sometimes the DataModel) and it would be nice if one of them was just `self`.

Reply:
I added an issue here: https://github.com/spacetelescope/stdatamodels/issues/80